examples/seismic/tti/operators.py

```diff
@@ -512,5 +526,116 @@ def AdjointOperator(model, geometry, space_order=4,
     return Operator(stencils, subs=model.spacing_map, name='AdjointTTI', **kwargs)
 
 
+def JacobianOperator(model, geometry, space_order=4,
+                     **kwargs):
+    """
+    Construct a Linearized Born operator in a TTI media.
+    Parameters
+    ----------
+    model : Model
+        Object containing the physical parameters.
+    geometry : AcquisitionGeometry
+        Geometry object that contains the source (SparseTimeFunction) and
+        receivers (SparseTimeFunction) and their position.
+    space_order : int, optional
+        Space discretization order.
+    kernel : str, optional
+        Type of discretization, centered or staggered.
+    """
+    dt = model.grid.stepping_dim.spacing
+    m = model.m
+    time_order = 2
+
+    # Create source and receiver symbols
+    src = Receiver(name='src', grid=model.grid, time_range=geometry.time_axis,
+                   npoint=geometry.nsrc)
+
+    rec = Receiver(name='rec', grid=model.grid, time_range=geometry.time_axis,
+                   npoint=geometry.nrec)
+
+    # Create wavefields and a dm field
+    u0 = TimeFunction(name='u0', grid=model.grid, save=None, time_order=time_order,
+                      space_order=space_order)
+    v0 = TimeFunction(name='v0', grid=model.grid, save=None, time_order=time_order,
+                      space_order=space_order)
+    du = TimeFunction(name="du", grid=model.grid, save=None,
+                      time_order=2, space_order=space_order)
+    dv = TimeFunction(name="dv", grid=model.grid, save=None,
+                      time_order=2, space_order=space_order)
+    dm = Function(name="dm", grid=model.grid, space_order=0)
+
+    # FD kernels of the PDE
+    FD_kernel = kernels[('centered', len(model.shape))]
+    eqn1 = FD_kernel(model, u0, v0, space_order)
+
+    # Linearized source and stencil
+    lin_usrc = -dm * u0.dt2
+    lin_vsrc = -dm * v0.dt2
+
+    eqn2 = FD_kernel(model, du, dv, space_order, qu=lin_usrc, qv=lin_vsrc)
+
+    # Construct expression to inject source values, injecting at u0(t+dt)/v0(t+dt)
+    src_term = src.inject(field=u0.forward, expr=src * dt**2 / m)
+    src_term += src.inject(field=v0.forward, expr=src * dt**2 / m)
+
+    # Create interpolation expression for receivers, extracting at du(t)+dv(t)
+    rec_term = rec.interpolate(expr=du + dv)
+
+    # Substitute spacing terms to reduce flops
+    return Operator(eqn1 + src_term + eqn2 + rec_term, subs=model.spacing_map,
+                    name='BornTTI', **kwargs)
+
+
+def JacobianAdjOperator(model, geometry, space_order=4,
+                        save=True, **kwargs):
+    """
+    Construct a linearized JacobianAdjoint modeling Operator in a TTI media.
```

Comment:
This is missing a blank line before "Parameters"

Reply:
the line was added.